service/src/main/kotlin/no/nav/su/se/bakover/service/søknadsbehandling/SøknadsbehandlingService.kt

```diff
@@ -30,6 +24,86 @@ interface SøknadsbehandlingService {
     fun hent(request: HentBehandlingRequest): Either<FantIkkeBehandling, Søknadsbehandling>
 }
 
+sealed class KunneIkkeLageBrevutkast {
+    data class KanIkkeLageBrevutkastForStatus(val status: BehandlingsStatus) : KunneIkkeLageBrevutkast()
+    object FantIkkeBehandling : KunneIkkeLageBrevutkast()
+    object KunneIkkeLageBrev : KunneIkkeLageBrevutkast()
+    object FantIkkePerson : KunneIkkeLageBrevutkast()
+    object FikkIkkeHentetSaksbehandlerEllerAttestant : KunneIkkeLageBrevutkast()
+}
+
+object FantIkkeBehandling
+
+sealed class KunneIkkeOppretteSøknadsbehandling {
+    object FantIkkeSøknad : KunneIkkeOppretteSøknadsbehandling()
+    object SøknadManglerOppgave : KunneIkkeOppretteSøknadsbehandling()
+    object SøknadErLukket : KunneIkkeOppretteSøknadsbehandling()
+    object SøknadHarAlleredeBehandling : KunneIkkeOppretteSøknadsbehandling()
+}
+
+sealed class KunneIkkeBeregne {
+    object FantIkkeBehandling : KunneIkkeBeregne()
+}
+
+sealed class KunneIkkeSimulereBehandling {
+    object KunneIkkeSimulere : KunneIkkeSimulereBehandling()
+    object FantIkkeBehandling : KunneIkkeSimulereBehandling()
+}
+
+sealed class KunneIkkeSendeTilAttestering {
+    object FantIkkeBehandling : KunneIkkeSendeTilAttestering()
+    object KunneIkkeFinneAktørId : KunneIkkeSendeTilAttestering()
+    object KunneIkkeOppretteOppgave : KunneIkkeSendeTilAttestering()
+}
+
+sealed class KunneIkkeUnderkjenneBehandling {
+    object FantIkkeBehandling : KunneIkkeUnderkjenneBehandling()
+    object AttestantOgSaksbehandlerKanIkkeVæreSammePerson : KunneIkkeUnderkjenneBehandling()
+    object KunneIkkeOppretteOppgave : KunneIkkeUnderkjenneBehandling()
+    object FantIkkeAktørId : KunneIkkeUnderkjenneBehandling()
+}
+
+sealed class KunneIkkeIverksetteBehandling {
+    object AttestantOgSaksbehandlerKanIkkeVæreSammePerson : KunneIkkeIverksetteBehandling()
+    object KunneIkkeUtbetale : KunneIkkeIverksetteBehandling()
+    object KunneIkkeKontrollsimulere : KunneIkkeIverksetteBehandling()
+    object SimuleringHarBlittEndretSidenSaksbehandlerSimulerte : KunneIkkeIverksetteBehandling()
+    object KunneIkkeJournalføreBrev : KunneIkkeIverksetteBehandling()
+    object FantIkkeBehandling : KunneIkkeIverksetteBehandling()
+    object FantIkkePerson : KunneIkkeIverksetteBehandling()
+    object FikkIkkeHentetSaksbehandlerEllerAttestant : KunneIkkeIverksetteBehandling()
+}
+
+sealed class KunneIkkeOppdatereBehandlingsinformasjon {
+    object FantIkkeBehandling : KunneIkkeOppdatereBehandlingsinformasjon()
+}
+
+data class KunneIkkeOppretteJournalpostForIverksetting(
```

Comment:
Mistenker at denne og opprettet jp for iverksetting egentlig hører hjemme sammen med "ferdigstilling"?